css/basscss.css

```diff
@@ -56,54 +68,17 @@ textarea {
   padding: .5rem .5rem;
 }
 
-.fieldset-reset {
+fieldset {
```

Comment:
I've added `.p0` and `.no-border` utilities. Planning on removing the base fieldset styles.